components/automate-cli/cmd/chef-automate/cmdUtils.go

```diff
@@ -0,0 +1,428 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/chef/automate/components/automate-deployment/pkg/cli"
+	"github.com/pkg/errors"
+)
+
+type Cmd struct {
+	PreExec   func(cmdInputs *CmdInputs, sshUtil SSHUtil, infra *AutomteHAInfraDetails, remoteService string, timestamp string, writer *cli.Writer) error
+	CmdInputs *CmdInputs
+}
+
+type CmdInputs struct {
+	Cmd         string
+	Args        []string
+	Single      bool
+	NodeIp      string
+	InputFiles  []string
+	Outputfiles []string
+	ErrorCheck  bool
```

Comment:
What does this actually mean? 
Is this for checking error in output ? then we can have:
```suggestion
	ErrorCheckEnableInOutput  bool
```